src/lib/mdns/Resolver.h

```diff
@@ -65,19 +65,22 @@ struct CommissionableNodeData
     bool IsValid() const { return !IsHost("") && ipAddress[0] != chip::Inet::IPAddress::Any; }
 };
 
-enum class CommissionableNodeFilterType
+enum class DiscoveryFilterType
```

Comment:
Add ` : uint8_t` to make `DiscoveryFilter` fit in 32-bit register?

Reply:
Great idea. Done. Thanks.